packages/fractal/fractal/src/fractal.test.js

```diff
@@ -123,15 +121,16 @@ describe('Fractal', function () {
       const fractal = makeFractal();
       const renderer = new Renderer(fractal.get('adapters'));
       const spy = sinon.spy(renderer, 'render');
-      const variant = parserOutput.components.first().getVariants().first();
+      const variant = parserOutput.components.first().getDefaultVariant();
       const view = parserOutput.components.first().getFiles().find({stem: 'view'});
       const opts = {
```

Comment:
As above - we should mock the renderer here

Reply:
Just added a commit that sorts this, stubs the methods rather than allowing a collections/renderers to be passed in as opts.